index.html

```diff
@@ -0,0 +1,152 @@
+        <p>The Internatiuonal Tango Festival will be held in Bogota, Colombia. The most famous milongueros from Colombia will present in this magnificent event.</p>
+      </div>
+      <div class="place">
+        <p class="date">2022.11.01 ~ 06 (monday to sunday)</p>
+        <p class="salon">@ Hotel Tequendama, Bogota , Colombia</p>
+      </div>
+    </section>
+
+    <main>
+      <!-- Program Section -->
+      <section id="program">
+        <!-- cross bg -->
+        <div class="cross-bg">
+          <h2 class="program-heading">Main Program</h2>
+          <div class="line"></div>
+
+          <!-- programs-container -->
+          <div class="programs-container">
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_01.png" alt="lecture icon">
+              <h3 class="program-card-heading">Workshop</h3>
+              <p class="program-desc">Dance and share with the best milongueros dancer from the world Don't miss it!.</p>
+            </div>
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_02.png" alt="exhibition icon">
+              <h3 class="program-card-heading">Milonga Class</h3>
+              <p class="program-desc">Dance and share with the best milongueros dancer from the world Don't miss it!.</p>
+            </div>
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_03.png" alt="forum icon">
+              <h3 class="program-card-heading">Tango Class</h3>
+              <p class="program-desc">Dance and share with the best milongueros dancer from the world Don't miss it!.</p>
+            </div>
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_04.png" alt="workshop icon">
+              <h3 class="program-card-heading">Vals Class</h3>
+              <p class="program-desc">Dance and share with the best milongueros dancer from the world Don't miss it!.</p>
+            </div>
+            <div class="program-card">
+              <img src="./src/img/icons/program_icon_05.png" alt="ignite icon">
+              <h3 class="program-card-heading">Milonga</h3>
+              <p class="program-desc">Dance and share with the best milongueros dancer from the world Don't miss it!.</p>
+            </div>
+          </div>
+          <!-- programs-container -->
+          <a class="program-link desktop" href="#">See the whole program</a>
+          <button class="program-button">
+            <a class="program-link mobile" href="#">Join International Tango Festival</a>
+          </button>
+        </div>
+        <!-- cross bg -->
+      </section>
+      <!-- Program Section -->
+
+      <!-- dancers section -->
+      <section id="dancers">
+        <h2 class="dancers-heading">Guest dancers</h2>
+        <div class="line"></div>
+        <div class="dancers"> </div>
+      </section>
```

Comment:
- The dancer's section of your project doesn't look exactly like the one in the design template.  This section should have 2 columns in a row of 3. You have aligned them in a single column in a row of 6. Please fix this issue 🙏. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

# Your project 👇:
-- Also for the mobile version, the 'description' text should be beside the dancer/speaker image, you have aligned it at the bottom of the image. Please try to fix this issue 👍 👍 
(Desktop version)    

<img width="948" alt="tango speakers" src="https://user-images.githubusercontent.com/73607512/197241011-2b835de5-7b7b-4ad9-afdf-371a86894e61.PNG">


(Mobile version) 

<img width="221" alt="tango speaker mobile" src="https://user-images.githubusercontent.com/73607512/197241449-b1461119-4a7e-46d2-a90c-ec5b5c45b848.PNG">


 # Design Template:

(Desktop version):

<img width="539" alt="speakers behance" src="https://user-images.githubusercontent.com/73607512/197242395-342f07cb-16a1-4075-87b5-5f97a574124a.PNG">


(Mobile version):
<img width="281" alt="speakers behance mobile" src="https://user-images.githubusercontent.com/73607512/197242594-6597558a-144b-4480-bef0-69c05e8602a3.PNG">

Reply:
ok